DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -79,6 +81,22 @@ public IActionResult Section(int customisationId, int sectionId)
             return View("Section/Section");
         }
 
+        [Route("/LearningMenu/{customisationId:int}/{sectionId:int}/{tutorialId:int}")]
+        public IActionResult Tutorial(int customisationId, int sectionId, int tutorialId)
+        {
+            var candidateId = User.GetCandidateId();
+            sessionService.StartOrUpdateSession(candidateId, customisationId, HttpContext.Session);

```

Comment:
@FridaTveit is this (line 88) something I should be doing here? I think I should but I thought I would check with you 🙂

Reply:
Should be doing it after the redirect to 404 🙂